lib/src/tests.rs

```diff
@@ -23,10 +23,15 @@ const DIGEST_SIZE: usize = 32;
 
 fn rand_tmp_dir() -> path::PathBuf {
     std::env::temp_dir().join("rdedup-tests").join(
-        rand::thread_rng()
+        match std::str::from_utf8(&rand::thread_rng()
```

Comment:
`.expect("must always be utf8").cloned()`?

Reply:
changed